fmriprep/workflows/base.py

```diff
@@ -338,9 +339,32 @@ def init_single_subject_wf(subject_id: str):
         skull_strip_fixed_seed=config.workflow.skull_strip_fixed_seed,
     )
 
+    # allow to run with anat-fast-track on fMRI-only dataset
+    if anatomical_cache and not subject_data['t1w']:
```

Comment:
I think we would want
```suggestion
    if 't1w_preproc' in anatomical_cache and not subject_data['t1w']:
```